_posts/2013-11-05-moveit-survey-results.markdown

```diff
@@ -4,18 +4,18 @@ comments: false
 date: 2013-11-05 08:15:12+00:00
 layout: post
 slug: moveit-survey-results
-title: MoveIt! Survey Results (2013)
-description: The results of the MoveIt! survey are in.
+title: MoveIt Survey Results (2013)
+description: The results of the MoveIt survey are in.
 redirect_from:
-  - "data/surveys/MoveIt!-2013-Survey.pdf"
+  - "data/surveys/MoveIt-2013-Survey.pdf"
```

Comment:
File needs to be renamed

Reply:
```suggestion
  - "data/surveys/MoveIt!-2013-Survey.pdf"
```
rather than break links, i'd rather leave the ! in for the filename